jaq-native/tests/tests.rs

```diff
@@ -188,6 +188,53 @@ yields!(
     json!([[1], 2, 3])
 );
 
+#[test]
+fn math() {
```

Comment:
Could you split up this function into multiple invocations of the yields! macro?  That way, the tests are getting shorter, and you should be able to frequently omit the json! macro.